libs/components/text-editor/src/lib/modules/text-editor/url-modal/text-editor-url-modal.component.html

```diff
@@ -10,7 +10,12 @@
     >
       <sky-tab tabHeading="Web page">
         <div class="sky-form-group sky-text-editor-url-modal-first-field">
-          <sky-input-box stacked="true">
+          <sky-input-box
+            stacked="true"
+            [hintText]="openLinksInNewWindowOnly ?
+              ('skyux_text_editor_url_modal_open_in_new_window_label'
+                | skyLibResources)
+             : null">
```

Comment:
The SKY UX team convention is to use `undefined` rather than `null`. We don't usually make a distinction between "variable/parameter hasn't been initialized/specified" and "this variable/parameter's value was set to a nullish value", so we try to stick with `undefined` when possible for consistency.

Reply:
Done. Changed from null to undefined.